app/models/exercise/challenge.rb

```diff
@@ -8,8 +8,10 @@ def reset!
     self.layout = self.class.default_layout
   end
 
+  alias_method :own_extra, :extra
+
   def extra(*)
-    [guide.extra, super]
+    [guide.extra, own_extra]
```

Comment:
I'm guessing that's the point, but this call changes as it's not passing arguments, is that alright?

Reply:
Actually this is a great point. I can't see any method call to extra which take a param. Is that splat needed altogether?